.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +1,33 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Something broken in SourceCred? Let us know!
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
 **Describe the bug**
-A clear and concise description of what the bug is.
+Please be brief & specific: 
 
-**To Reproduce**
-Steps to reproduce the behavior:
+**Steps to reproduce**
+What sequence of actions results in this error?  e.g.:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
 **Expected behavior**
-A clear and concise description of what you expected to happen.
+What you expected to happen instead of the bug.
 
 **Screenshots**
-If applicable, add screenshots to help explain your problem.
+Please add screenshots of your problem with a brief description of each (alt text). 
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
-
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser [e.g. stock browser, safari]
- - Version [e.g. 22]
+**Technical Details:**
+These are important!
+ - OS & version: [e.g. Catalina 10.15.7]
+ - Browser [e.g. chrome, safari, check "about" in your browser menu for version info]
+ - SC Instance [e.g. SC, 1Hive, etc]
```

Comment:
nit: Maybe expand “SC” to “SourceCred” here for clarity?

Reply:
Two additional notes here: 
* Do we want to specifically request the browser version?
* I think we should also request the link to the instance on line 30.

Reply:
Good points, integrating these.